docs/enterprise/concepts.md

```diff
@@ -106,6 +106,10 @@ Routes define the connection pathway and configuration from the internet to your
 
 This term refers to the system or service the route provides or restricts access to.
 
+### Moving Routes
+
+When moving a Route from one [Namespace](#namespace) to another, enforced policies will automatically be removed or applied. Optional policies available in the source Namespace but not the target will prevent the move. This is intentional, to prevent unassociated policies.
```

Comment:
```suggestion
When moving a Route from one [Namespace](#namespace) to another, enforced policies will automatically be removed or applied. Optional policies available in the source Namespace but not the target will prevent the move. This is intentional to prevent unassociated policies.
```

I think there is not comma, but I am not sure. 

> If the independent clause comes first, no punctuation separates the two.

Reply:
I think it's a style choice in this example, but I tend to over-comma in my style so happy to adjust. Thanks!